content/docs/analytics/GoogleAnalytics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Integrate Google Analytics'
+meta_title: 'Implementing Appbase analytics - Core Concepts'
+meta_description: 'A short guide on how to setup google analytics and track search terms and queries.'
+keywords:
+    - concepts
+    - appbase
+    - google-analytics
+    - integrate
+sidebar: 'docs'
+---
+
+##Setting Up Google Analytics
+Even though ReactiveSearch provides an analytics tailored to your search, you might want to integrate your search into your existing analytics tools.
+The base library components doesn't provide a built-in widget to implement analytics with other providers.
+
+Integrating with Google Analytics requires 2 steps:
+
+-   set up the Google Analytics library in your page
+-   setup the search state change listener
+
+To set up Google Analytics, the best way is to actually [follow the reference guide](https://developers.google.com/analytics/devguides/collection/analyticsjs/).
+
+Once the GA library is installed on your website, look below how Google Analytics can be integrated with different libraries
+
+##Integrating With Libraries
```

Comment:
Title should be: "Setup The Search State Change Listener" based on the two steps you highlighted above. If this isn't accurately worded, also change the referenced bullet point's text.

Reply:
Add the following line before starting the library specific sections.

"Setting the search state change listener depends on how your search app is setup. If you are using one of the appbase.io libraries, you can refer to the sections below on how you can do this.